mautic/mautic
# Added character counter and information text to sms new form

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ no]
| New feature/enhancement? (use the a.x branch)      | [ yes]
| Deprecations?                          | [ no]
| BC breaks? (use the c.x branch)        | [ no]
| Automated tests included?              | [ no] 
| Related user documentation PR URL      | https://github.com/mautic/mautic-documentation/pull/257
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
Considering the fact that sms are limited to 160 characters.
I added a counter and an information notice  to the sms creation form to inform the users on how many characters they are currently using.




#### Steps to test this PR:


1. Load up [this PR](https://mautibox.com)
2. Create a new sms
3. Type something in the textaera 
4. See that counter increases
5. See that the information box is present under the textaera


![image](https://user-images.githubusercontent.com/50580844/136513999-80c7065c-0a9a-45e4-8fc9-624c6a690bb8.png)


<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10514"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
